charts/rancher-wins-upgrader/templates/_helpers.tpl

```diff
@@ -6,6 +6,14 @@
 {{- end -}}
 {{- end -}}
 
+{{- define "windows_prefix_path" -}}
+{{ ternary "C:/" .Values.global.cattle.rkeWindowsPrefixPath "" }}
+{{- end -}}
+
+{{- define "windows_host_path" -}}
+{{ include "windows_prefix_path" . | replace "C:/" "C:/host/" }}
```

Comment:
I have two questions here...

1. We always prepend with `/host`?
2. What if `C:/` is not provided?

Reply:
Do we support having this installed on a different drive at all? Have we tested using a different root drive? I did this in the node cleanup and given that it is all based on windows specific values I just assumed. Good call out.

Reply:
We should force the `C:/` drive. I am more concerned about handling whatever the user puts in `windows_prefix_path` :)

Reply:
@nickgerace  I updated it and did a force push so you may have to dig for that change.